x/stake/keeper/validator.go

```diff
@@ -105,19 +80,19 @@ func (k Keeper) getAllValidators(ctx sdk.Context) (validators Validators) {
 }
 
 // Get the set of all validators, retrieve a maxRetrieve number of records
-func (k Keeper) GetValidators(ctx sdk.Context, maxRetrieve int16) (validators Validators) {
+func (k Keeper) GetValidators(ctx sdk.Context, maxRetrieve int16) (validators []types.Validator) {
 	store := ctx.KVStore(k.storeKey)
 	iterator := sdk.KVStorePrefixIterator(store, ValidatorsKey)
 
-	validators = make([]Validator, maxRetrieve)
+	validators = make([]types.Validator, maxRetrieve)
 	i := 0
 	for ; ; i++ {
 		if !iterator.Valid() || i > int(maxRetrieve-1) {
 			iterator.Close()
```

Comment:
Can this be closed outside of the loop, per #1125

Reply:
addressed